src/pfe/portal/routes/projects/remoteBind.route.js

```diff
@@ -231,15 +231,17 @@ router.post('/api/v1/projects/:id/upload/end', async (req, res) => {
           filesToDelete.map(oldFile => cwUtils.forceRemove(path.join(pathToTempProj, oldFile)))
         );
         res.sendStatus(200);
-
+ 
+        await cwUtils.copyProject(pathToTempProj, path.join(project.workspace, project.directory), getMode(project));
```

Comment:
Looks like a good change - I don't know how metrics injection was working previously, when this line was in `syncToBuildContainer`. (How did we inject metrics before the project was copied from `cw-temp` to `codewind-workspace`?)